### ISSUE TITLE
Singleplayer world freezes when loaded for a second time

### ISSUE THREAD

**Issue by pfmoore** [FTBTeam/FTB-Continuum#396]
* **Modpack Version**: 1.4.1

* **Issue**: When I start my instance (single player world) from MultiMC, it loads fine and I can play. However, when I exit the world to the menu screen, and then try to restart it (via "Single Player", select the world, and start it) the world opens, and I get the "Generating terrain" message, then the "0%" message, but the game freezes there. It never actually loads. I can hit "Escape" to go back to the main menu, and then select "Quit" to exit, but when my game goes back to the MultiMC loader, it still shows as running (even though the Minecraft window is closed) and the only way I have of quitting is to kill the instance. The linked log file comes from having killed the instance in this way, but hopefully some of the earlier parts of the log indicate what caused the freeze.

The following section looks suspicious:

```
[17:36:34] [Server thread/ERROR] [FML]: A fatal exception occurred during the server starting event
net.minecraftforge.fml.common.LoaderExceptionModCrash: Caught exception from Immersive Engineering (immersiveengineering)

Caused by: java.lang.IllegalStateException: A JEI API method is being called by another mod from the wrong thread. It must be called on the main thread.
```

* **Link to Log or Crash File Paste**: https://paste.ee/p/jmeBA

* **Is it Repeatable?**: Yes, happens pretty much every time.

* **Mod/s Affected**: N/A

* **Known Fix**: No fix known, although I can get back into my game by killing the instance and restarting it as described above.

**Comment by Proha5** [comment]
same problem

**Comment by Icebarker** [comment]
Same for me but it also says the same for Journeymap. In issue #356 manually updating IE seemed to help me on my test world and I can load it up no problem but my main world still does it. Might help you but I think there is a bigger issue

**Comment by TheFrostSeraph** [comment]
same issue here

**Comment by Isentiototus** [comment]
Just started happening to me today as well.  If I load any world that is it, if I exit it I cannot re-enter any worlds including newly made one.  It works again once I Kill it as per OP

**Comment by CaveJohnsonn** [comment]
I fixed this by updating Immersive Engineering.

**Comment by abesto** [comment]
Looks like the same issue as https://github.com/FTBTeam/FTB-Presents-Direwolf20-1.12/issues/29. The release notes for the release fixing the issue on Direwolf do include upgrading Immersive Engineering from the version used in Continuum (0.12-83 -> 0.12-84). Confirming the upgrade fixed this for me as well in Continuum.

**Comment by tfox83** [comment]
Fixed in 1.5.0
